setup.cfg

```diff
@@ -222,3 +222,123 @@ type = layers
 layers=
     napari.qt
     napari.components
+
+[conda_installer]
+# NOTE: packages in this section follow the conda-install syntax
+# *not* the pip or poetry ones, which might be slightly different!
+# If you edit this config, please leave a blank line between blocks
+# to help the shell tooling do its work <3
+
+# dependencies to build the conda packages (nightlies) right
+build_packages =
+    conda-souschef
+    conda-smithy
+    python 3.8
+    pip
+    tomlkit
+    ruamel.yaml
+    anaconda-client
+
+build_packages_channels =
+    conda-forge
+
+# run requirements in the napari conda package
+# this needs to be in sync with pypi's, a bit above
+napari_recipe_run =
```

Comment:
@jaimergp we could perhaps add a test that checks that this list (conda) and the one above (pip) are in sync.

This should help us keep things synced. 

I do see a lot of value in keeping all these pinnings in a single file (even if duplicate) than putting them in a separate `environment.yaml`